common/v2/features/Dashboard/components/ActionTile.scss

```diff
@@ -67,3 +67,13 @@
     }
   }
 }
+
+.wallet-icon {
+  width: 35px;
+  height: 35px;
+
+  @media (min-width: 680px) {
```

Comment:
we want to have a common place for all breakpoints and media queries otherwise it becomes painful to control.
Since you are in scss, use one of the variables in `common/sass/variables/grid.scss`